course_catalog/task_helpers.py

```diff
@@ -245,8 +251,9 @@ def digest_ocw_course(
         is_published (Bool): Flags OCW course as published or not
         course_prefix (String): (Optional) String used to query S3 bucket for course raw JSONs
```

Comment:
Since we're modifying this function, could we add `Returns:` while we're at it? Also some of the args types are not valid python types (`Bool` -> `bool`, `String` -> `str`)

Reply:
This function doesn't return anything, but I updated the types